speakers/src/main/java/in/droidcon/speakers/ui/list/SpeakerListFragment.kt

```diff
@@ -1,20 +1,21 @@
-package `in`.droidcon.speakers.presentation.ui
```

Comment:
Why does info module implementation PR has speakers module changes? Can we have these speakers module changes as separate PR?